uber/models.py

```diff
@@ -1805,6 +1805,8 @@ def track_pageview(cls, url, query):
             if "registration" in url:
                 with Session() as session:
                     attendee = session.query(Attendee).filter(Attendee.id == params['id']).first()
+                    if (attendee == None):
+                        raise HTTPRedirect('../registration/error_attendee_not_found')
```

Comment:
Looks good! Two minor things about this code:

1) if-statements in Python don't need parenthesis.

2) Although ``if attendee == None`` is indeed valid, the idiomatic way to compare to ``None`` in Python is ``if attendee is None``.  That's just a style thing though, since both versions do basically the same thing in this case (the latter being slightly more efficient).